src/Playwright.Tests/PageRequestContinueTests.cs

```diff
@@ -127,5 +127,26 @@ public async Task ShouldNotThrowWhenContinuingAfterPageIsClosed()
             await tsc.Task;
             await done;
         }
+
+        [PlaywrightTest("page-request-continue.spec.ts", "should not allow changing protocol when overriding url")]
+        public async Task ShouldNotAllowChangingProtocolWhenOverridingUrl()
+        {
+            PlaywrightException exception = null;
```

Comment:
Adjust to https://github.com/microsoft/playwright/pull/11701/files which is similar to what I had before I have seen the test in JavaScript.